website/common/script/ops/buy/buyMysterySet.js

```diff
@@ -1,14 +1,18 @@
-import i18n from '../i18n';
-import content from '../content/index';
+import i18n from '../../i18n';
+import content from '../../content/index';
 import get from 'lodash/get';
 import each from 'lodash/each';
 import {
   BadRequest,
   NotAuthorized,
   NotFound,
-} from '../libs/errors';
+} from '../../libs/errors';
+
+module.exports = function buyMysterySet (user, req = {}, analytics, _window) {
+  if (typeof window !== 'undefined') {
```

Comment:
is this used for tests only? What about definind a dummy window under `global.window` in tests?

Reply:
oh thanks for the hint!, thats much better